scripts/lib/setup/installers.sh

```diff
@@ -319,6 +345,30 @@ function use_android_sdk() {
 }
 
 function install_android_ndk() {
+  if is_macos; then
+    install_android_ndk_mac
+  elif is_linux; then
+    install_android_ndk_linux
+  fi
+}
+
+function install_android_ndk_linux() {
+  if [ -e ~/Android/android-ndk-r10e/ ]; then
```

Comment:
Maybe we could test if the folder already exists at `$ANDROID_NDK` too (maybe the user has it instaled in another location and already has `ANDROID_NDK` defined)?